src/controllers/space-controller.js

```diff
@@ -15,6 +15,10 @@ function SpaceController(el, parent) {
   //   // is a brand new Space component.
   //   window.location.reload();
   // }
+  var self = this,
+    reinitialize = _.debounce(function () {
+      self.init();
+    }, 100);
 
   this.el = el;
 
```

Comment:
I'm pretty sure `this.spaceRef` isn't being used in this file anywhere. Could you delete that?